test.js

```diff
@@ -0,0 +1,53 @@
+const accessSync = require('fs').accessSync
```

Comment:
I was planning on just running them manually at each go. I don't think I release often enough to require CI/CD.